docs/source/tutorials/GEDI/GEDI_L4A_Example.ipynb

```diff
@@ -0,0 +1,583 @@
+{
```

Comment:
<p>You can remove <code>lat_lowestmode</code> and <code>lon_lowestmode</code> from the list of columns. Since those are specified via the <code>lat</code> and <code>lon</code> inputs, those 2 columns are used to construct the <code>geometry</code> column that is automatically added to the resulting GeoDataFrame written to the output file. In your code farther below, rather than extracting the <code>lat_lowestmode</code> and <code>lon_lowestmode</code> columns from the output file, get the <code>geometry</code> attribute of the GeoDataFrame.        </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Thank you will remove those from columns list. I forgot to use <code>geometry</code> attribute. Will update accordingly.  </p>